### PR TITLE
Better display footnote links and table

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: static/js/apps/eval_rig/util.ts [hunk 1]
```diff
@@ -40,4 +40,11 @@ export const processText = (text: string): string => {
       return `<span class="annotation annotation-${callId}">${innerHtml}</span>`;
     }
   );
+  // Format the links
+  const httpPattern = /https:\/\/[^\s]+/g;
```
Comment (chejennifer) [inline static/js/apps/eval_rig/util.ts:44]: nit: can make this a global constant outside the function?

**Review by chejennifer — APPROVED** [review]
nice!

File: static/js/apps/eval_rig/util.ts (hunk 1 above)
Comment (shifucun) [reply static/js/apps/eval_rig/util.ts]: Done

**Comment by shifucun** [comment]
thanks for review!

**Review by pradh — APPROVED** [review]

**Pull request merged**